instrument/REF_L_Parameters.xml

```diff
@@ -11,9 +11,28 @@
       <value val="304"/>
     </parameter>
 
+    <!-- Pixel size is in mm -->
     <parameter name="pixel-width">
       <value val="0.7"/>
     </parameter>
+
+    <parameter name="sample-det-distance">
+      <value val="1.83"/>
+    </parameter>
+
+    <!-- Effective distances used in the reduction, accounting for emission time -->
```

Comment:
[The delayed emission time is wavelength dependent](https://docs.mantidproject.org/nightly/algorithms/ModeratorTzero-v1.html) :thinking:

Reply:
Yes. It's a small correction and we found that taking an average gave good results and was better than having no offset. We may think of a better way to model this in the future.